### PR TITLE
WIP: Pelumi/update/sdk docs

### PR DESCRIPTION
**Pull request by Oluwadaminiola**
this pr is for the updated design for SDKs and the app portal

### CODE REVIEW

**Comment by horlah** [comment]
Screenshot 2022-03-22 at 6 28 56 PM

The links in this section are leading to the SaaS. @Oluwadaminiola

File: web/ui/website/assets/images/sdk.svg [hunk 1]
```diff
@@ -0,0 +1,9 @@
...
+</defs>
+</svg>
```
Comment (horlah) [inline web/ui/website/assets/images/sdk.svg:9]: Please replace this with a png image. Base 64 string in SVG images takes longer to load most times, and considering they'll likely open this on a phone or tab.

**Review by horlah — CHANGES_REQUESTED** [review]

**Comment by horlah** [comment]
Screenshot 2022-03-22 at 8 44 53 PM

Please check the space here against JK's design. @Oluwadaminiola

File: web/ui/website/layouts/docs.vue [hunk 2]
```diff
@@ -19,13 +19,13 @@
...
 				<li v-for="(page, index) in pages" :key="index">
-					<template v-if="page.id !== 'guide' && page.id !== 'index'">
+					<template v-if="page.id !== 'guide' && page.id !== 'index' && page.id !== 'convoy.js' && page.id !== 'convoy-python' && page.id !== 'convoy-php'">
 						<nuxt-link :to="'/docs/' + page.id">
 							<img src="~/assets/images/angle-down-icon.svg" alt="angle right" />
 							{{ page.title }}
 						</nuxt-link>
 
-						<ul v-if="page.toc.length > 0">
+						<ul v-if="page.toc.length > 0" class="" :class="{ show: currentPage == page.id }">
```
Comment (horlah) [inline web/ui/website/layouts/docs.vue:28]: I'm not sure I understand why these sub-links here are hidden. @Oluwadaminiola

File: web/ui/website/layouts/docs.vue [hunk 3]
```diff
@@ -34,6 +34,12 @@
 						</ul>
 					</template>
 				</li>
+				<li>
+					<nuxt-link to="/docs/sdk">
+						<img src="~/assets/images/angle-down-icon.svg" alt="angle right" />
+						SDK
```
Comment (horlah) [inline web/ui/website/layouts/docs.vue:40]: Hmm, nice hack here. We should check it out.

File: web/ui/website/pages/docs/index.vue [hunk 4]
```diff
@@ -5,36 +5,40 @@
...
+						</a>
 					</div>
 				</div>
+				<div class="banner-img">
+					<img src="~/assets/images/doc-banner.svg" alt="banner" />
+				</div>
+			</div>
+			<!-- <section class="cards cards__banner">
+				
+			</section> -->
```
Comment (horlah) [inline web/ui/website/pages/docs/index.vue:29]: This is no longer needed right?

**Review by horlah — COMMENTED** [review]
Cool stuff here @Oluwadaminiola, I think we should review the logic for the SDK page.

File: web/ui/website/assets/images/sdk.svg (hunk 1 above)
Comment (Oluwadaminiola) [reply web/ui/website/assets/images/sdk.svg:9]: okay noted

File: web/ui/website/pages/docs/index.vue [hunk 5]
```diff
@@ -57,6 +60,29 @@
...
+					</p>
+					<nuxt-link to="/docs/app-portal">
+						<button>
+							Read more
+							<img src="~/assets/images/dark-arrow-circle-right.svg" alt="arrow right" />
+						</button>
+					</nuxt-link>
+				</div>
+
+				<!-- <div class="cards_item pink">
```
Comment (horlah) [inline web/ui/website/pages/docs/index.vue:85]: Please delete this @Oluwadaminiola

**Review by horlah — APPROVED** [review]
Please rename this PR before merging

**Pull request merged**